extension/scripts/global/globalClasses.js

```diff
@@ -67,7 +67,7 @@ class FeatureManager {
 
 		document.find(".tt-page-status-header").onclick = function () {
 			this.classList.toggle("collapsed");
-			ttStorage.change({ filters: { containers: { [containerID]: this.classList.contains("collapsed") } } });
+			ttStorage.change({ filters: { containers: { [this.containerID]: this.classList.contains("collapsed") } } });
```

Comment:
Did you test this? Wouldn't `this` go to the element instead of the class?

Reply:
Just pushed a fix for this.